docs/examples/template.csv

```diff
@@ -1,5 +1,5 @@
-ID,Label,Definition,Definition Source,See Also,Editor,RDF Type,Class Type,Parent IRI
-ID,A rdfs:label,A IAO:0000115,A IAO:0000119,AI rdfs:seeAlso,A IAO:0000117,TYPE,CLASS_TYPE,CI
-ex:F344N,F 344/N,An inbred strain of rat used in many scientific investigations.,James A. Overton,http://www.informatics.jax.org/external/festing/rat/docs/F344.shtml,James A. Overton,owl:Class,subclass,NCBITaxon:10116
-ex:B6C3F1,B6C3F1,An inbred strain of mouse used in many scientific investigations.,James A. Overton,http://jaxmice.jax.org/strain/100010.html,James A. Overton,owl:Class,subclass,NCBITaxon:10090
-ex:rat-1234,rat 1234,,,F 344/N,,ex:F344N,,
```

Comment:
This should not need to be changed. If it does, then something has broken backwards compatibility.

Reply:
Fixed to old version

Reply:
This is not the original version. The [original version](https://github.com/ontodev/robot/blob/master/docs/examples/template.csv) has a `CI` pattern that the new code does not accept:

```
UNKNOWN TEMPLATE ERROR could not interpret template string "CI" for column 9 ("Parent IRI") in table "template.csv".
```

So the new code is breaking backwards compatibility.